app/templates/crate/version.hbs

```diff
@@ -104,7 +104,7 @@
             {{/if}}
         {{/if}}
     </div>
-    <div class='authorship'>
+    <section class='authorship' aria-label="Authorship">
```

Comment:
This class being named "authorship" is kind of... out of date. There's a lot more metadata in this crate than just the author. Not sure what a better label would be- "Crate info"? "Crate metadata"?